### ISSUE TITLE
explore possiblity of making an importly web-app

### ISSUE THREAD

**Issue by chase-moskal** [chase-moskal/importly#1]
where you can paste in a `package.json` file and it will spit out an `import-map`

i think the browser can even send out the GET requests for the `package.json`'s and crawl around for stuff :+1:

**Comment by chase-moskal** [comment]
i wonder if importly will need to generate its own importmap in order to load itself on its own webapp?

hah

this task is a low priority for me

**Comment by chase-moskal** [comment]
this should be easy after #7 is happy
